core-foundation/src/propertylist.rs

```diff
@@ -86,8 +86,9 @@ pub trait CFPropertyListSubClass: TCFType {
 
 impl CFPropertyListSubClass for ::data::CFData {}
 impl CFPropertyListSubClass for ::string::CFString {}
-impl CFPropertyListSubClass for ::array::CFArray {}
-impl CFPropertyListSubClass for ::dictionary::CFDictionary {}
+impl<T> CFPropertyListSubClass for ::array::CFArray<T> {}
+impl<K, V> CFPropertyListSubClass for ::dictionary::CFDictionary<K, V> {}
+impl<K, V> CFPropertyListSubClass for ::dictionary::CFMutableDictionary<K, V> {}
```

Comment:
I'm not completely sure this is correct or safe, but I think it is. The docs say that a mutable dictionary can be viewed as an immutable one, from my understanding. But not the other way around. So looking at a mutable dictionary as a propertylist should just be an extension of viewing it as an immutable dictionary then?